modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -297,4 +301,72 @@ default <U> ObservableValue<U> flatMap(Function<? super T, ? extends ObservableV
     default ObservableValue<T> when(ObservableValue<Boolean> condition) {
         return new ConditionalBinding<>(this, condition);
     }
+
+    /**
+     * Creates a {@link Subscription} on this {@code Observable} which calls the given
+     * {@code changeSubscriber} with the old and new value whenever its value changes.
+     * <p>
+     * The parameters supplied to the {@link BiConsumer} are the old and new value
+     * respectively.
+     * <p>
+     * Note that the same subscriber instance may be safely subscribed for
+     * different {@code Observables}.
+     * <p>
+     * Also note that when subscribing on an {@code Observable} with a longer
+     * lifecycle than the subscriber, the subscriber must be unsubscribed
+     * when no longer needed as the subscription will otherwise keep the subscriber
+     * from being garbage collected. Considering creating a derived {@code ObservableValue}
+     * using {@link #when(ObservableValue)} and subscribing on this derived observable value
+     * to automatically decouple the lifecycle of the subscriber from this
+     * {@code ObservableValue} when some condition holds.
+     *
+     * @param changeSubscriber a {@code BiConsumer} to supply with the old and new values
+     *     of this {@code ObservableValue}, cannot be {@code null}
+     * @return a {@code Subscription} which can be used to cancel this
+     *     subscription, never {@code null}
+     * @throws NullPointerException if the subscriber is {@code null}
+     * @see #addListener(ChangeListener)
+     * @since 21
+     */
+    default Subscription subscribe(BiConsumer<? super T, ? super T> changeSubscriber) {
+      Objects.requireNonNull(changeSubscriber, "changeSubscriber cannot be null");
+      ChangeListener<T> listener = (obs, old, current) -> changeSubscriber.accept(old, current);
+
+      addListener(listener);
+
+      return () -> removeListener(listener);
+    }
+
+    /**
+     * Creates a {@link Subscription} on this value which immediately provides
+     * the current value to the given {@code valueSubscriber}, followed by any
+     * subsequent values whenever its value changes.
+     * <p>
+     * Note that the same subscriber instance may be safely subscribed for
+     * different {@code Observables}.
+     * <p>
+     * Also note that when subscribing on an {@code Observable} with a longer
+     * lifecycle than the subscriber, the subscriber must be unsubscribed
+     * when no longer needed as the subscription will otherwise keep the subscriber
+     * from being garbage collected. Considering creating a derived {@code ObservableValue}
+     * using {@link #when(ObservableValue)} and subscribing on this derived observable value
+     * to automatically decouple the lifecycle of the subscriber from this
+     * {@code ObservableValue} when some condition holds.
```

Comment:
I think that these explanations are no longer needed if you add something like what I suggested for `Subscription`. It covers the lifecycle and multiple subscriptions explanations already.